sc_preprocessing/2_make_pseudobulks/pseudobulk_synovium.ipynb

```diff
@@ -0,0 +1,1192 @@
+{
```

Comment:
<p>Are these commented out on purpose?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>removed in <a href="https://github.com/greenelab/buddi_analysis/pull/12/commits/6a0f2690a72842aece33e7952b8fdc09bc9efa06" target="_blank">6a0f269</a></p>